specification/agfood/resource-manager/readme.csharp.md

```diff
@@ -11,5 +11,5 @@ csharp:
   clear-output-folder: true
   client-side-validation: false
   namespace: Microsoft.AgFoodPlatform
```

Comment:
Do we want to keep the namepace as `Microsoft.AgriFoodPlatform`

Reply:
updated to Microsoft.Azure.Management.AgriFood